src/components/button.js

```diff
@@ -35,7 +35,6 @@ module.exports = function(app) {
               case 'custom':
               case 'oauth':
               case 'url':
-                return;
```

Comment:
Is there a reason we are removing the return? This will override the button type.

Reply:
This was a cleanup item from PR https://github.com/formio/ngFormio/commit/54736e550e5a16fbeab140540ca6e867c20ed829

having that return causes every custom button on the platform to submit the form

Reply:
sounds good. thanks.